### ISSUE TITLE
publicPath does not work with URL

### ISSUE THREAD

**Issue by f0x52** [typedoc2md/typedoc-plugin-markdown#688]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

When configuring `publicPath` with an url (like the [docs](https://typedoc-plugin-markdown.org/docs/options/utility-options#--publicpath) suggest), the internal call to `path.resolve()` normalizes the `//` in the `http(s)://` protocol bit to just `/`, resulting in broken links
```markdown
[Reference](https:/example.com/README.md#reference)
```

I think https://github.com/typedoc2md/typedoc-plugin-markdown/issues/621 was about the same problem

### TypeDoc configuration

```json
{
  "publicPath": "https://example.com"
}
```

### Expected behavior

Currently `publicPath` has the correct behavior for setting a path prefix, but should preserve the protocol when a full URL is given

**Comment by f0x52** [comment]
Typedoc also has options for [hostedBaseUrl](https://typedoc.org/options/output/#hostedbaseurl) and [useHostedBaseUrlForAbsoluteLinks](https://typedoc.org/options/output/#usehostedbaseurlforabsolutelinks) but I believe those are never used for markdown output?

**Comment by tgreyuk** [comment]
Thankss. This should be fixed in typedoc-plugin-markdown@4.2.8

> Typedoc also has options for [hostedBaseUrl](https://typedoc.org/options/output/#hostedbaseurl) and [useHostedBaseUrlForAbsoluteLinks](https://typedoc.org/options/output/#usehostedbaseurlforabsolutelinks) but I believe those are never used for markdown output?

This is correct. These options were introduced after the `publicPath` plugin option was implemented. I have considered supporting but I think the single option is more simplistic. Have updated the docs to reflect this.
